book/website/communication/citable/citable-cite.md

```diff
@@ -45,6 +45,10 @@ You can find examples of these statements in the publishers' (research data) aut
 -  “The data that support the findings of this study are available upon request.
 Access conditions and procedures can be found at [URL to restricted access repository such as [EASY](https://easy.dans.knaw.nl/ui/home).]”
 
+**More Data Availability Statement examples:**
+
+You can find more examples on the [Manchester's Data Access statements page](https://www.library.manchester.ac.uk/using-the-library/staff/research/research-data-management/sharing/data-access-statements/), the [Cambridge Data Availability Statement examples](https://www.cambridge.org/core/services/authors/open-data/data-availability-statements), the [AMS Data Availability Statement examples](https://www.ametsoc.org/index.cfm/ams/publications/author-information/formatting-and-manuscript-components/data-availability-statement-examples/), or [Nature's Tips for writing a dazzling Data Availability Statement](https://researchdata.springernature.com/posts/tips-for-writing-a-dazzling-das-data-availability-statement).
```

Comment:
Likewise, I would use "other" in place of "more here".

A recommended change would be to capitalise "S":

- "Manchester's Data Access statements" → "Manchester's Data Access Statements"

Also, should it say Manchester University?

Reply:
Thanks! I'll capitalise the S. I tossed the universities out there because otherwise this sentence becomes super long :P